projects/plugins/jetpack/extensions/blocks/subscriptions/panel.js

```diff
@@ -65,12 +81,7 @@ export default function SubscribePanels() {
 	const showNotices = Number.isFinite( subscriberCount ) && subscriberCount > 0;
 	return (
 		<>
-			{ isNewsletterFeatureEnabled() && (
-				<PluginDocumentSettingPanel title={ __( 'Newsletter', 'jetpack' ) }>
-					<NewsletterAccess setPostMeta={ setPostMeta } accessLevel={ accessLevel } />
-				</PluginDocumentSettingPanel>
-			) }
-
+			<AccessLevelSelectorPanel setPostMeta={ setPostMeta } accessLevel={ accessLevel } />
```

Comment:
This is what I meant earlier, if that makes sense:
```suggestion
			<AccessLevelSelectorPanel setPostMeta={ setPostMeta } accessLevel={ accessLevel } />
			 
			  { ! postWasEverPublished && (
			    
			    [ rest of the file ]
			  
			  )}
```
But it was only how I would do it...